digdag-tests/src/test/java/acceptance/td/TdResultExportIT.java

```diff
@@ -0,0 +1,148 @@
+package acceptance.td;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Optional;
+import com.treasuredata.client.TDApiRequest;
+import com.treasuredata.client.TDClient;
+import com.treasuredata.client.TDClientConfig;
+import com.treasuredata.client.model.TDExportResultJobRequest;
+import com.treasuredata.client.model.TDJob;
+import com.treasuredata.client.model.TDSaveQueryRequest;
+import com.treasuredata.client.model.TDSavedQuery;
+import com.treasuredata.client.model.TDSavedQueryStartRequest;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static acceptance.td.Secrets.TD_API_KEY;
+import static acceptance.td.Secrets.TD_API_ENDPOINT;
+import static org.hamcrest.Matchers.isEmptyOrNullString;
+import static org.hamcrest.Matchers.not;
+import static org.junit.Assume.assumeThat;
+
+public class TdResultExportIT
```

Comment:
I may be confused, but it seems there is no test for new td_result_export> operator.
In my understand, integration test directly run the new functionality with TemporaryDigdagServer in most cases.
Could you refer another Td operators tests in same directory ?
It will be helpful for you.